<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>ExportHTMLDialog</class>
  <widget class="QDialog" name="ExportHTMLDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>634</width>
        <height>510</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>0</width>
        <height>130</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Export HTML</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
        <layout class="QHBoxLayout">
          <property name="spacing">
            <number>6</number>
          </property>
          <property name="margin">
            <number>0</number>
          </property>
          <item>
            <widget class="QLabel" name="textLabel1">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="minimumSize">
                <size>
                  <width>125</width>
                  <height>0</height>
                </size>
              </property>
              <property name="text">
                <string>Export to directory:</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="lineEditDir">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="minimumSize">
                <size>
                  <width>300</width>
                  <height>0</height>
                </size>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="browseExportDirButton">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Browse</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="minimumSize">
            <size>
              <width>0</width>
              <height>222</height>
            </size>
          </property>
          <property name="maximumSize">
            <size>
              <width>16777215</width>
              <height>16777214</height>
            </size>
          </property>
          <property name="title">
            <string>Options</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
              <rect>
                <x>20</x>
                <y>20</y>
                <width>581</width>
                <height>173</height>
              </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
              <item>
                <widget class="QCheckBox" name="imageButton">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Image of map</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="TOCButton">
                  <property name="text">
                    <string>Table of contents</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="taskFlagsButton">
                  <property name="text">
                    <string>Task flags</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="userFlagsButton">
                  <property name="text">
                    <string>General flags</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="numberingButton">
                  <property name="text">
                    <string>Number sections</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="textColorButton">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Colored headings in text</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QCheckBox" name="saveSettingsInMapButton">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Save settings in map</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="groupBox_3">
          <property name="minimumSize">
            <size>
              <width>0</width>
              <height>100</height>
            </size>
          </property>
          <property name="title">
            <string>Stylesheets</string>
          </property>
          <widget class="QWidget" name="layoutWidget1^">
            <property name="geometry">
              <rect>
                <x>20</x>
                <y>30</y>
                <width>499</width>
                <height>78</height>
              </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
                <widget class="QCheckBox" name="copyCssButton">
                  <property name="text">
                    <string>Copy CSS from </string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QLineEdit" name="lineEditCssSrc">
                  <property name="minimumSize">
                    <size>
                      <width>300</width>
                      <height>0</height>
                    </size>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <widget class="QLineEdit" name="lineEditCssDst"/>
              </item>
              <item row="1" column="0">
                <widget class="QLabel" name="label">
                  <property name="text">
                    <string>Use CSS from</string>
                  </property>
                  <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                </widget>
              </item>
              <item row="0" column="2">
                <widget class="QPushButton" name="browseCssSrcButton">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Browse</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="2">
                <widget class="QPushButton" name="browseCssDstButton">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Browse</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
            <size>
              <width>0</width>
              <height>70</height>
            </size>
          </property>
          <property name="title">
            <string>Scripts</string>
          </property>
          <widget class="QWidget" name="layoutWidget2">
            <property name="geometry">
              <rect>
                <x>10</x>
                <y>20</y>
                <width>505</width>
                <height>26</height>
              </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
                <widget class="QLabel" name="textLabel2_2">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="minimumSize">
                    <size>
                      <width>125</width>
                      <height>0</height>
                    </size>
                  </property>
                  <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="text">
                    <string>After Export:</string>
                  </property>
                  <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLineEdit" name="lineEditPostScript">
                  <property name="minimumSize">
                    <size>
                      <width>300</width>
                      <height>0</height>
                    </size>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="browsePostExportButton">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Browse</string>
                  </property>
                </widget>
              </item>
            </layout>
          </widget>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <spacer>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
                <size>
                  <width>61</width>
                  <height>21</height>
                </size>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="cancelButton">
              <property name="text">
                <string>Cancel</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="exportButton">
              <property name="text">
                <string>Export</string>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="local">xsltproc.h</include>
    <include location="local">vymprocess.h</include>
  </includes>
  <resources/>
  <connections>
    <connection>
      <sender>cancelButton</sender>
      <signal>clicked()</signal>
      <receiver>ExportHTMLDialog</receiver>
      <slot>reject()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>459</x>
          <y>443</y>
        </hint>
        <hint type="destinationlabel">
          <x>256</x>
          <y>233</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>exportButton</sender>
      <signal>clicked()</signal>
      <receiver>ExportHTMLDialog</receiver>
      <slot>accept()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>368</x>
          <y>443</y>
        </hint>
        <hint type="destinationlabel">
          <x>256</x>
          <y>233</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>
